chirp/wxui/main.py

```diff
@@ -815,7 +815,7 @@ def _editor_page_changed(self, event):
         self._editors.GetPage(event.GetSelection())
         self._update_window_for_editor()
         radio = self.current_editorset.radio
-        radio_name = '%s %s%s' % (radio.VENDOR, radio.MODEL, radio.VARIANT)
+        radio_name = '%s %s %s' % (radio.VENDOR, radio.MODEL, radio.VARIANT)
```

Comment:
So, the problem with this is that some drivers have things like `R` or `E` as their variant, for European or American versions of the radio, which will make this look like `Yaesu FT-2D R` which is.. odd.

I assume you don't need to see the variant information in your case, you just don't want it to look squashed? I would say the best thing for us to do is just drop the variant out of this string entirely as `Yaesu FT-2D` is better than the alternative, especially since most of the drivers don't even put the `R` or `E` in the model string at all. You okay with that?

Reply:
>the best thing for us to do is just drop the variant out of this string entirely
>

Removing the variant from the status bar would be the sort of compromise where everyone suffers equally. A much better solution is just do nothing! That is,  drop the patch and wait for someone to find a better solution.